project/src/main/java/com/google/sps/data/README

```diff
@@ -0,0 +1 @@
+This directory is a placeholder for backend Java classes and other data.
```

Comment:
Can you please move all the files under ...com/google/sps/ensemble/...? This is to ensure that your namespace does not collide with others also in the com/google/sps namespace.